sql/commands/sql-alter-source.mdx

```diff
@@ -173,7 +173,15 @@ ALTER SOURCE source_name
     SET SOURCE_RATE_LIMIT { TO | = } { default | rate_limit_number };
 ```
 
-Use this statement to modify the rate limit of a source. For the specific value of `SOURCE_RATE_LIMIT`, refer to [How to view runtime parameters](/docs/current/view-configure-runtime-parameters/#how-to-view-runtime-parameters).
+Use this statement to modify the rate limit of a source. For the specific value of `SOURCE_RATE_LIMIT`, refer to [How to view runtime parameters](/operate/view-configure-runtime-parameters/#how-to-view-runtime-parameters).
+
+<Note>
+For a newly created materialized view on a shared source, it will backfill from
```

Comment:
The concept "shared source" is opaque to users. Recommend to use

```suggestion
For a newly created materialized view on a source with historical data e.g. Kafka source, it will backfill from
```